docker-compose.yml

```diff
@@ -14,13 +24,15 @@ services:
       - 8390:8390
     environment:
       - PATH_PREFIX=/api
-  nginx_proxy:
-    # Child services must have links named "ui", "api".
-    image: nginx
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2
     ports:
-      - 4400:4400
-    volumes:
-      - ./nginx.conf:/etc/nginx/nginx.conf
-    links:
-      - ui
-      - api
+      - 9200:9200
+  kibana:
+    image: docker.elastic.co/kibana/kibana-oss:6.2.2
+    ports:
+      - 5601:5601
+  # For convenience, load test data into Elasticsearch.
+  index_test_data:
```

Comment:
How do you plan to swap out indexers here?

Reply:
I will add instructions later. It will look like:
- Before running Data explorer, write index to some directory.
- Somehow point Elasticsearch to that directory.
- Somehow point to dataset config directory which has dataset name.

FYI each index will have a different name and an Elasticsearch instance can have many indices. So there could be a test index, baseline index, onduo index, etc. I figure I can write the "test" index every time since it's fast. It's ok if it's not used.